python/lib/dependabot/python/file_updater/pip_compile_file_updater.rb

```diff
@@ -483,6 +490,32 @@ def pip_compile_options_from_compiled_file(requirements_file)
           options
         end
 
+        def uv_pip_compile_options_from_compiled_file(requirements_file)
+          options = ["--output-file=#{requirements_file.name}"]
+
+          options << "--no-emit-index-url" unless requirements_file.content.include?("index-url http")
+
+          options << "--generate-hashes" if requirements_file.content.include?("--hash=sha")
+
+          options << "--no-annotate" unless requirements_file.content.include?("# via ")
+
+          options << "--pre" if requirements_file.content.include?("--pre")
+
+          options << "--no-strip-extras" if requirements_file.content.include?("--no-strip-extras")
+
+          if requirements_file.content.include?("--no-binary") || requirements_file.content.include?("--only-binary")
+            options << "--emit-build-options"
+          end
+
+          if (resolver = RESOLVER_REGEX.match(requirements_file.content))
+            options << "--resolver=#{resolver}"
+          end
```

Comment:
I don't think there's any reason to pass this to uv, since [it's just there for better UX](https://github.com/astral-sh/uv/blob/23b010c08f8f500ec2bd21076f53c361b6251c49/crates/uv-cli/src/compat.rs#L91-L104)